cmd/crowdsec-cli/cliallowlists/allowlists.go

```diff
@@ -0,0 +1,562 @@
+	if err != nil {
+		cobra.CompError("unable to load dbclient " + err.Error())
+		return nil, cobra.ShellCompDirectiveNoFileComp
+	}
+
+	allowlists, err := cli.db.ListAllowLists(ctx, false)
+	if err != nil {
+		cobra.CompError("unable to list allowlists " + err.Error())
+		return nil, cobra.ShellCompDirectiveNoFileComp
+	}
+
+	ret := []string{}
+
+	for _, allowlist := range allowlists {
+		if strings.Contains(allowlist.Name, toComplete) && !slices.Contains(args, allowlist.Name) {
+			ret = append(ret, allowlist.Name)
+		}
+	}
+
+	return ret, cobra.ShellCompDirectiveNoFileComp
+}
+
+func (cli *cliAllowLists) listHuman(out io.Writer, allowlists *models.GetAllowlistsResponse) {
+	t := cstable.NewLight(out, cli.cfg().Cscli.Color).Writer
+	t.AppendHeader(table.Row{"Name", "Description", "Creation Date", "Updated at", "Managed by Console", "Size"})
+
+	for _, allowlist := range *allowlists {
+		t.AppendRow(table.Row{allowlist.Name, allowlist.Description, allowlist.CreatedAt, allowlist.UpdatedAt, allowlist.ConsoleManaged, len(allowlist.Items)})
+	}
+
+	io.WriteString(out, t.Render()+"\n")
+}
+
+func (cli *cliAllowLists) listContentHuman(out io.Writer, allowlist *models.GetAllowlistResponse) {
+	t := cstable.NewLight(out, cli.cfg().Cscli.Color).Writer
+	t.AppendHeader(table.Row{"Value", "Comment", "Expiration", "Created at"})
+
+	for _, content := range allowlist.Items {
+		expiration := "never"
+		if !time.Time(content.Expiration).IsZero() {
+			expiration = content.Expiration.String()
+		}
+		t.AppendRow(table.Row{content.Value, content.Description, expiration, allowlist.CreatedAt})
+	}
+
+	io.WriteString(out, t.Render()+"\n")
+}
+
+func (cli *cliAllowLists) NewCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:               "allowlists [action]",
+		Short:             "Manage centralized allowlists",
+		Args:              cobra.MinimumNArgs(1),
+		DisableAutoGenTag: true,
+	}
+
+	cmd.AddCommand(cli.newCreateCmd())
+	cmd.AddCommand(cli.newListCmd())
+	cmd.AddCommand(cli.newDeleteCmd())
+	cmd.AddCommand(cli.newAddCmd())
+	cmd.AddCommand(cli.newRemoveCmd())
+	cmd.AddCommand(cli.newInspectCmd())
+	return cmd
+}
+
+func (cli *cliAllowLists) newCreateCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "create [allowlist_name]",
+		Example: "cscli allowlists create my_allowlist -d 'my allowlist description'",
+		Short:   "Create a new allowlist",
+		Args:    cobra.ExactArgs(1),
+		PersistentPreRunE: func(cmd *cobra.Command, _ []string) error {
+			var err error
+			cfg := cli.cfg()
+			if err = require.LAPI(cfg); err != nil {
+				return err
+			}
+			cli.db, err = require.DBClient(cmd.Context(), cfg.DbConfig)
+			if err != nil {
+				return err
+			}
+			return nil
+		},
+		RunE: cli.create,
+	}
+
+	flags := cmd.Flags()
+
+	flags.StringP("description", "d", "", "description of the allowlist")
+
+	cmd.MarkFlagRequired("description")
+
+	return cmd
+}
+
+func (cli *cliAllowLists) create(cmd *cobra.Command, args []string) error {
```

Comment:
I think we want console-based and cscli-based allowlists to have different prefixes ?